cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -39,29 +46,49 @@ type ProbeEvent struct {
 
 // addBlockDeviceEvent fill block device details from different probes and push it to etcd
 func (pe *ProbeEvent) addBlockDeviceEvent(msg controller.EventMessage) {
-	bdList, err := pe.Controller.ListBlockDeviceResource()
+	bdAPIList, err := pe.Controller.ListBlockDeviceResource(true)
```

Comment:
What is a `bdAPIList`?  is it `existingBDs` ?

Reply:
yes, the list of `BlockDevice` resources in the cluster.